IPInfo.IO/ipinfo/trigger_fetch_ipinfo_database.py

```diff
@@ -0,0 +1,272 @@
+import ipaddress
+import json
+import time
+import gzip
+import logging
+import uuid
+from datetime import datetime, timedelta
+from functools import cached_property
+from io import BytesIO
+from ipaddress import IPv6Network, IPv4Network
+from typing import Iterator
+from iso3166 import countries
+
+import requests
+from sekoia_automation.trigger import Trigger
+
+
+class TriggerFetchIPInfoDatabase(Trigger):
+    MAX_HOUR_TAG_VALID_FOR: int = 10 * 24  # Tags are valid for 10 days
+
+    @cached_property
+    def api_token(self):
+        return self.configuration.get("IPINFO_TOKEN", "42039d1a8e5800")
```

Comment:
Should be from the module configuration also this is required, we should not specify a fallback value.

**Also you just gave you API token in a public repo. You should revoke it as soon as possible.**